File: Client/Assets/iTweenExamples/AccurateLob/Scripts/AccurateLob.cs
using UnityEngine;
using System.Collections;

public class AccurateLob : MonoBehaviour
{
	public Transform target;
	public GameObject bomb;

	void Update ()
	{
		RaycastHit hit = new RaycastHit();
		Ray cameraRay = Camera.main.ScreenPointToRay(Input.mousePosition);
		if (Physics.Raycast (cameraRay.origin,cameraRay.direction,out hit, 100)) {
			target.position = hit.point;
		}
		
		if (Input.GetMouseButtonDown(0)) {
			GameObject newBomb = (GameObject)Instantiate(bomb);
			Bomb newBombScript = newBomb.GetComponent<Bomb>();
			newBombScript.targetPosition = target.position;
		}
	}
}

File: Client/Assets/Package/Pixelplacement/iTweenExamples/AccurateLob/Scripts/AccurateLob.js
var target : Transform;
var bomb : GameObject;

function Update (){
	var hit : RaycastHit = new RaycastHit();
	var cameraRay : Ray  = Camera.main.ScreenPointToRay(Input.mousePosition);
	if (Physics.Raycast (cameraRay.origin,cameraRay.direction,hit, 100)) {
		target.position = hit.point;
	}
	
	if (Input.GetMouseButtonDown(0)) {
		var newBomb : GameObject = Instantiate(bomb);
		var newBombScript : Bomb = newBomb.GetComponent(Bomb);
		newBombScript.targetPosition = target.position;
	}
}